tasks/install.deb.yml

```diff
@@ -2,6 +2,22 @@
 
 - include_vars: "{{ansible_distribution}}.yml"
 
```

Comment:
Not it should be present before installation or it fails to start after package install